doc/lsst.pipe.base/task-framework-overview.rst

```diff
@@ -0,0 +1,49 @@
+.. NOTE: this topic should be edited into the modernized topic-based documentation style.
+
+.. _task-framework-overview:
+
+##############################
+Overview of the task framework
+##############################
+
+:doc:`lsst.pipe.base <index>` provides a data processing pipeline infrastructure.
+Data processing is performed by **tasks**, which are instances of `lsst.pipe.base.Task` or `lsst.pipe.base.CmdLineTask`.
+Tasks perform a wide range of data processing operations, from basic operations such as assembling raw images into CCD images (trimming overscan), fitting a WCS or detecting sources on an image, to complex combinations of such operations.
```

Comment:
You have to do more than just trim overscan -- there's no way to avoid estimating its level too.  This isn't the point you're trying to make, but the parenthetical comment is confusing.  I think it'd be better to use the full ISR as the example.